api/src/api/users/user_schemas.py

```diff
@@ -681,3 +686,55 @@ class UserInvitationListResponseSchema(AbstractResponseSchema):
         fields.Nested(UserInvitationItemSchema),
         metadata={"description": "List of user invitations"},
     )
+
+
+class UserResponseOrgInvitationRequestSchema(Schema):
+    status = fields.Enum(
+        OrganizationInvitationStatus,
+        validate=validators.OneOf(
+            [
+                OrganizationInvitationStatus.ACCEPTED,
+                OrganizationInvitationStatus.REJECTED,
+            ]
+        ),
+        metadata={"description": "User response to invitation"},
+    )
+
+
+class OrganizationSchema(Schema):
+    organization_id = fields.UUID(
+        metadata={
+            "description": "The unique identifier of the organization",
+            "example": "456e7890-e12c-34f5-b678-901234567890",
+        }
+    )
+    organization_name = fields.String(
+        allow_none=True,
+        metadata={
+            "description": "Organization name",
+            "example": "Legal business name of the corresponding Sam Gov Entity",
+        },
+    )
+
+
+class InvitationRoleSchema(Schema):
+    role_id = fields.UUID(metadata={"description": "Role unique identifier"})
+    role_name = fields.String(metadata={"description": "Role name"})
+
+
+class OrganizationInvitationSchema(Schema):
+    organization_invitation_id = fields.UUID(metadata={"description": "Organization invitation ID"})
+    status = fields.Enum(
+        OrganizationInvitationStatus, metadata={"description": "User response to invitation"}
+    )
+    responded_at = fields.DateTime(metadata={"description": "Time User responded to invitation"})
+    organization = fields.Nested(
+        OrganizationSchema, metadata={"description": "Organization information"}
+    )
+    roles_granted = fields.List(
+        fields.Nested(InvitationRoleSchema), metadata={"description": "Roles granted"}
+    )
```

Comment:
```suggestion
    roles_granted = fields.List(
        fields.Nested(InvitationRoleSchema), metadata={"description": "Roles granted"}, attribute="roles"
    )
```

This way you dont have to specify roles_granted explicitly in the response